CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## Unreleased
+
+### Changed
+
+ - improved and sped up testing (first steps towards [#139](https://github.com/GIScience/ohsome-py/issues/139))
```

Comment:
The change probably didn’t significantly affect testing speed. It only made it easier to conduct.

Reply:
very slight speed up, will leave it :-)